### PR TITLE
doc/cephadm: Disable automatic deployment of daemons

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Fixes: https://tracker.ceph.com/issues/45767
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/cephadm/operations.rst [hunk 1]
```diff
@@ -355,3 +355,65 @@ the following command on the host:
...
+.. _cephadm-spec-unmanaged:
+
+Disable automatic deployment of daemons
+=======================================
+
+Cephadm supports disabling the automated deployment and removal of daemons per service. In
+this case, the CLI supports two commands that are dedicated to this mode. 
+
+In order to disable the automatic management of dameons, apply
+the :ref:`orchestrator-cli-service-spec` with ``unamanged=True``.
```
Comment (jmolmo) [inline doc/cephadm/operations.rst:369]: typo: /unamanged/unmanaged

**Review by jmolmo — APPROVED** [review]
LGTM (after fix the typo)

File: doc/cephadm/operations.rst (hunk 1 above)
Comment (zdover23) [reply doc/cephadm/operations.rst]: This is a nitpick, but I'd remove "In order" and just begin the sentence with "To disable".

Comment (sebastian-philipp) [reply doc/cephadm/operations.rst]: fixed

**Pull request merged**